### ISSUE TITLE
Tela de edição de usuário para colocar os horários

### ISSUE THREAD

**Issue by lamenezes** [PyLadiesSanca/PyLorean#7]

**Comment by ariehfabbro** [comment]
Eu faço, perguntinha: Essa tela eu faço com bootstrap?

**Comment by lamenezes** [comment]
sim, tudo em bootstrap: http://getbootstrap.com. O bootstrap já vem com os estilos prontos, é mais fácil. Sem contar que o template já está encaminhado e feito em bootstrap

Essa tela (issue #7) depende da issue #3. Sem os models (banco de dados) não tem como fazer a tela de cadastro horários. 

Outra coisa: @fcschmidt o template está pronto para usar? Vc está com a issue #2. O @ariehfabbro precisa disso para mexer nisso

**Comment by fcschmidt** [comment]
ele não depende da minha parte.
Pois as telas são independentes

**Comment by lamenezes** [comment]
depende do base.html, arquivo que contém o layout padrão do site do site, para que ele já faça a tela usando o design q vc criou.

**Comment by fcschmidt** [comment]
Verdade. 
Criei um arquivo base.html, vou atualizar ele hoje e a noite eu subo.
